docs/source/daml/reference/interfaces.rst

```diff
@@ -18,6 +18,19 @@ method) are implemented. This allows decoupling such behavior from its
 implementation, so other developers can write applications in terms of the
 interface instead of the concrete template.
 
+Configuration
+*************
+In order to test this experimental feature, you need to enable
+the dev mode on both Daml and Canton.
+
+For Daml, you need to add in ``daml.yaml`` the following line:
+.. code-block:: none
+
+   build-options: [--target=1.dev]
+
+To enable dev mode for Canton, follow the instructions in [this documentation](https://www.canton.io/docs/dev/user-manual/usermanual/FAQ.html#how-do-i-enable-unsupported-features)
```

Comment:
Can you add an anchor in the Canton repo and use `:ref` to link to that to ensure that we get a compile error if this link goes stale and so that it links to docs.daml.com?

Reply:
ok thanks